### PR TITLE
Add StringToBins() and NewFromBins() functions

### PR DESCRIPTION
**Pull request by redhotpenguin**
Looking for feedback on these changes to deserialize the DecStrings() output back into a []Bin. The use case is to take the output of DecStrings(), send it over the wire, and combine those bins with others into an aggregate histogram.

The deserialization code is a bit crude; if it looks like it was written by a Perl programmer, there's a reason for that. I used split rather than slicing the string as bytes as the implementation seemed to be easier to read, though the byte slice approach would have the advantage of one less dependency (and probably a bit faster).

Please let me know how this looks; pretty sure it can be improved on.

### CODE REVIEW

**Comment by redhotpenguin** [comment]
Need to add a test here to make sure that the new histogram can be composed of bins correctly; I suspect I might need to merge bins that cover the same range.

**Comment by redhotpenguin** [comment]
Closing this in favor of a different approach. This is really a single use case, though the effort was educational.

**Comment by postwait** [comment]
Sorry I didn't get to review this, but it seems useful.

The only thing I might say is that those function should be private and a NewFromStrings be the only thing exposed.

**Comment by redhotpenguin** [comment]
No problem thanks for the review. Updated PR coming soon.

File: circonusllhist_test.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 	"time"
 
-	hist "github.com/circonus-labs/circonusllhist"
+	hist "github.com/redhotpenguin/circonusllhist"
```
Comment (postwait) [inline circonusllhist_test.go]: can't pull with that.

**Comment by redhotpenguin** [comment]
Sorry, thoughtless error. Was going to drop a note saying this is ready to go but wasn't quite ready. Should be ready to go now.

I had considered putting the stringsToBin private method test in circonusllhist.go as suggested by a go-nuts thread, but that would require the testing package also, which didn't seem great. I think there's a minimum account of test coverage on NewFromStrings to make reasonably sure it's doing what it should be.

**Pull request merged**